splice/web/api/campaign.py

```diff
@@ -126,5 +131,40 @@ def put(self, campaign_id):
 api.add_resource(CampaignAPI, '/campaigns/<int:campaign_id>', endpoint='campaign')
 
 
+_VALID_CREATIVE_EXTENSIONS = set(['zip'])
+_VALID_ASSETS_EXTENSIONS = set(['tsv', 'txt'])
+
+
+@campaign_bp.route('/campaigns/<int:campaign_id>/bulkupload', methods=['POST'])
+def handler_bulk_upload(campaign_id):
+    """bulk uploading adgroups and tiles with the given campaign_id."""
+
+    def is_allowed_files(name, allowed_sets):
+        return '.' in name and name.rsplit('.', 1)[1] in allowed_sets
+
+    creatives = request.files["creatives"]
+    assets = request.files["assets"]
+    if not is_allowed_files(creatives.filename.lower(), _VALID_CREATIVE_EXTENSIONS) \
+            or not is_allowed_files(assets.filename.lower(), _VALID_ASSETS_EXTENSIONS):
```

Comment:
why don't you do the `.filename.lower()` inside `is_allowed_files()`? So you pass the file to `is_allowed_files()` and it checks the extension. That would clean this up a little. 

And I think it should be `is_allowed_file()` without the `s` since you only check one file at a time.